src/Uno.UI/UI/Xaml/Controls/ListViewBase/BufferViewCache.Android.cs

```diff
@@ -282,7 +283,7 @@ private void TrimEmpty()
 		/// <summary>
 		/// Prefetch views in the target range that aren't yet in the buffer.
 		/// </summary>
-		private void PrefetchViews(RecyclerView.Recycler recycler)
+		private void PrefetchViews(RecyclerView.Recycler recycler, RecyclerView.State state)
```

Comment:
What's this `state` and why do we have to propagate it everywhere ? Seems to be tightly linked to the `recycler`, wondering if this state could not be <kept in | packed with> the `recycler`?

Reply:
It's something we're passed by the `RecyclerView`, it contains the current source state (no. of items etc), whereas `Recycler` contains the machinery for recycling items. We could propagate them together as a tuple but it comes to pretty near the same amount of code.